### PR TITLE
docs: update accordion group template [skip chromatic]

### PR DESCRIPTION
**Pull request by auroraVasconcelos**
## Description:
Closes https://github.com/solid-design-system/solid/issues/1441

## Definition of Reviewable:

- [x] Documentation is created/updated
- [ ] Migration Guide is created/updated
- [ ] E2E tests (features, a11y, bug fixes) are created/updated

- [x] Stories (features, a11y) are created/updated
- [ ] relevant tickets are linked

### CODE REVIEW

**Review by karlbaumhauer — APPROVED** [review]

**Comment by MartaPintoTeixeira** [comment]
1 ) Please sync the titles with FIGMA:
Accordion Group Background White
Accordion Group Background Neutral-100
Accordion Group Background Primary-100

2 ) This copy: Examples of the sd-accordion-group component in different backgrounds.
Should be after the title

**Review by MartaPintoTeixeira — APPROVED** [review]

File: packages/placeholders/src/images/financialNetwork.jpg
Comment (mariohamann) [inline packages/placeholders/src/images/financialNetwork.jpg:1]: Move this out of the placeholder package and instead assets of Storybook!

File: packages/components/src/templates/accordion-group.stories.ts [hunk 1]
```diff
@@ -88,29 +120,47 @@ export const NeutralBackground = {
...
+              class="w-full aspect-ratio"
+              src="./placeholders/images/coins.jpg"
+              alt="Close-up of hands stacking coins into small piles on a table, suggesting financial planning or saving."
+            />
+          </figure>
+        </sd-accordion>
+        <sd-accordion summary="Investment philosophy">
           <div class="sd-prose">
-            <p>
+            <p class="sd-paragraph">
```
Comment (mariohamann) [inline packages/components/src/templates/accordion-group.stories.ts:155]: Why do you need sd-paragraph here, when there is already sd-prose? Maybe it's more that we need to remove the Storybook styles e. g. in a decorator somewhere?

File: packages/placeholders/src/images/graph.jpg
Comment (karlbaumhauer) [inline packages/placeholders/src/images/graph.jpg:1]: same as above (move into storybook asstets) or get this approved as placeholder by brand.

**Review by karlbaumhauer — CHANGES_REQUESTED** [review]

File: packages/placeholders/src/images/financialNetwork.jpg
Comment (auroraVasconcelos) [reply packages/placeholders/src/images/financialNetwork.jpg:1]: Updated

File: packages/placeholders/src/images/graph.jpg
Comment (auroraVasconcelos) [reply packages/placeholders/src/images/graph.jpg:1]: Updated

**Comment by karlbaumhauer** [comment]
@auroraVasconcelos can you pls update your branch from main? For some reasons, the images you removed should already be removed from my PR #1547 but still showing here... :/

**Review by Vahid1919 — APPROVED** [review]
Besides the existing comments, LGTM.

File: packages/components/src/templates/accordion-group.stories.ts (hunk 1 above)
Comment (auroraVasconcelos) [reply packages/components/src/templates/accordion-group.stories.ts]: Updated

**Comment by auroraVasconcelos** [comment]
> @auroraVasconcelos can you pls update your branch from main? For some reasons, the images you removed should already be removed from my PR #1547 but still showing here... :/

Updated

**Review by karlbaumhauer — APPROVED** [review]

**Pull request merged**